cmake/configs/default.cmake

```diff
@@ -255,35 +262,39 @@ hunter_default_version(eigen3-nnls VERSION 1.0.1)
 hunter_default_version(enet VERSION 1.3.13-p1)
 hunter_default_version(entityx VERSION 1.3.0-p1)
 hunter_default_version(eos VERSION 0.12.1)
+hunter_default_version(etc2comp VERSION 0.0.0-9cd0f9c-p0)
 hunter_default_version(ethash VERSION 0.5.0)
 hunter_default_version(farmhash VERSION 1.1)
+hunter_default_version(fast_obj VERSION 1.0-p0)
 hunter_default_version(ffmpeg VERSION n4.1-dev-45499e557c-p7)
 hunter_default_version(fft2d VERSION 1.0.0-p0)
+hunter_default_version(filament VERSION 1.9.8)
 hunter_default_version(fixesproto VERSION 5.0)
-hunter_default_version(flatbuffers VERSION 1.8.0-p1)
+hunter_default_version(flatbuffers VERSION 1.12.0)
 hunter_default_version(flex VERSION 2.6.4)
-hunter_default_version(fmt VERSION 6.1.2)
+hunter_default_version(fmt VERSION 7.1.2)
```

Comment:
FYI, this will break `soralog` with following error:
```
/build/source/src/impl/sink_to_console.cpp: In function 'void soralog::{anonymous}::put_reset_style(char*&)':
/build/source/src/impl/sink_to_console.cpp:57:32: error: 'fmt::internal' has not been declared
   57 |       const auto &style = fmt::internal::data::reset_color;
```

Reply:
Branch master contains commits with fixed soralog.